### ISSUE TITLE
Provide placeholder for absolute path

### ISSUE THREAD

**Issue by steini86** [nextcloud/workflow_script#86]
All my scripts work with absolute paths and I use something like `/var/www/m.js86.de/files/ocdata/"$1"` where the first parameter is %n. However, it should be easy to directly provide a placeholder with the absolute path. (If I understand it correctly, %n is created by using the full path and stripping the base).

**Comment by blizzz** [comment]
> (If I understand it correctly, %n is created by using the full path and stripping the base)

nope, it's more complicated: https://github.com/nextcloud/workflow_script/blob/9fe9d443d34c825aab2173074802381c0eb61d86/lib/Operation.php#L265-L331

%f should give you the absolute path for files available on local storage. Those from external will be copied to a temp directory.
